cognito-pre-token-generation-openid/src/main/java/no/unit/nva/cognito/UserSelectionUponLoginHandler.java

```diff
@@ -111,6 +111,15 @@ public UserSelectionUponLoginHandler(CognitoIdentityProviderClient cognitoClient
     @Override
     public CognitoUserPoolPreTokenGenerationEvent handleRequest(CognitoUserPoolPreTokenGenerationEvent input,
                                                                 Context context) {
+        try {
```

Comment:
Loggene som tilhører alarmen inneholder kun melding om customer not found exception og tidsbruk, som Ketil korrekt sier skal kun logges som warning. Sjekkes også tidsbruken på lambdaen, den så også grei ut. Så da er det fortsatt et mysterium hvorfor den trynet. Forhåpentligvis kan dette gi oss mer informasjon.